### ISSUE TITLE
Merge multiple PDFDocuments

### ISSUE THREAD

**Issue by techprimate-phil** [techprimate/TPPDF#67]
As mentioned in #66 the ability of merging two `PDFDocuments` into one resulting PDF file should be considered.

Multiple documents get calculated. Then during the render process, all calculated objects are in one list which will then be rendered into a single file.

Known issues:

- Pagination will not work → should not work in some cases?
- How to implement a generic solution?

**Comment by jakelindsay** [comment]
Is this still being considered? I am currently using a workaround to convert pdf pages from other documents into uiimages, and then compiling them into the tppdf. I can get it to work in the simulator, but on the iPhone 8 the app crashes due to memory limitations.

**Comment by techprimate-phil** [comment]
Even tough this issue is over a year old, I will reconsider it now, due to personal needs.

For now I'll take the approach simply combining the calculated pages and render it in a single run through, using something like:

```swift
try PDFGenerator.generateURL(documents: [
    document1, 
    document2
], filename: "ChatBook.pdf", progress: { value in
    print(value)
})
```

**Comment by techprimate-phil** [comment]
WIP on feature/multi-docs

**Comment by techprimate-phil** [comment]
Released in 1.5.1
